src/util-thash.c

```diff
@@ -320,6 +329,14 @@ THashTableContext* THashInit(const char *cnf_prefix, size_t data_size,
     return ctx;
 }
 
+/* \brief Set memcap to current memuse
+ * */
+void THashConsolidateMemcap(THashTableContext *ctx)
+{
+    ctx->config.memcap = SC_ATOMIC_GET(ctx->memuse);
```

Comment:
wondering if we should update this to make the memcap something like `MAX(SC_ATOMIC_GET(ctx->memuse), default_memcap)`. If the configs memcap is higher than the memuse of the loaded set, use the memcap setting instead.

Reply:
Makes sense.